CHANGELOG.md

```diff
@@ -1,117 +1,138 @@
+## 12.0.0
+
+_Released `2022-03-23`_
+
+We're breaking slightly from our pattern of a single yearly major release. The `12.0.0` release contains some backwards-incompatible changes to how middleware and auth refreshes work. For the most part, you'll be able to upgrade to this version safely, but as always, it's worth re-running unit tests (especially those related to authentication).
```

Comment:
Tried to add a little context here and let them know we're doing another major version soonish.